db/schema.rb

```diff
@@ -10,12 +10,34 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 2021_04_13_153428) do
+ActiveRecord::Schema.define(version: 2021_04_15_154339) do
+
+  create_table "attended_events", force: :cascade do |t|
+    t.integer "user_id", null: false
+    t.integer "event_id", null: false
+    t.datetime "created_at", precision: 6, null: false
+    t.datetime "updated_at", precision: 6, null: false
+    t.index ["event_id"], name: "index_attended_events_on_event_id"
+    t.index ["user_id"], name: "index_attended_events_on_user_id"
+  end
+
+  create_table "events", force: :cascade do |t|
+    t.string "location"
+    t.datetime "date"
+    t.datetime "created_at", precision: 6, null: false
+    t.datetime "updated_at", precision: 6, null: false
+    t.integer "user_id", null: false
```

Comment:
- [ ] The user should be mentioned at the events table with the `creator_id` column name instead of `user_id`.